framework/include/eventloop/eventloop.h

```diff
@@ -199,18 +221,18 @@ el_event_t *eventloop_add_event_handler(int type, event_callback func, void *cb_
  * @brief Delete registered handler for event
  * @details @b #include <eventloop/eventloop.h> \n
  * The registered event handler is deleted and all used resources for event handler are freed.
- * @param[in] handle a pointer of event handle to be deleted
+ * @param[in] event_handler a pointer of event handle to be deleted
```

Comment:
As you revised description of `el_event_t`, desciption for the return of `eventloop_add_event_handler` also need to be revised.